compiler/modules/nback/build.bal

```diff
@@ -222,7 +222,7 @@ function buildConvertRepr(llvm:Builder builder, Scaffold scaffold, Repr sourceRe
         }
     }
     // this shouldn't ever happen I think
-    return scaffold.unimplementedErr("unimplemented conversion required");
+    return scaffold.unimplementedErr("unimplemented conversion required", pos);
```

Comment:
I think it's easier to turn this into

```
panic err:impossible("unexpected ...");
```

Reply:
And then we don't need to bother with passing around the position.

Reply:
fixed